### DISCUSSION TITLE
Invoices. How to create the templates?

### DISCUSSION THREAD

**Discussion by WhiteTiger-IT** [kimai/kimai#2336] [General]
If I try to create an invoice, the message "Please create an invoice template first" always appears.
However, I didn't understand how I can create the templates.
I've created the var/invoices folder.
Now do I have to copy the contents of template/invoice here?
Also the sub-folder "renderer"?
How do I load and modify the other templates?

**Comment by kevinpapst** [comment]
If you see `Please create an invoice template first` you should see a form below as well.
Enter the details into the form and save, this is your first invoice template.

Don't get confused by the wording. Invoice template here means the settings for an invoice type. The layout itself is a file (called invoice document). Save the form and you can afterwards select entries to generate an invoice.

**Reply by WhiteTiger-IT** [reply]
OK. Now I have created two templates, for an invoice and for a timesheet. However, I did not understand where they were saved.
However under the templates folder I see many other; for example I see one with the name Freelance, another with the name company.docx.
Where do I load them to use them?
To customize them, I need to edit the ones in the templates folder or copy them to the var/invoices folder.

**Reply by vic-t** [reply]
The creation and customization of invoice templates is actually very well explained in the documentation: 

https://www.kimai.org/documentation/invoices.html#create-your-own-invoice-document

If you're going to create a Word based template, you can actually upload and use it right from the Kimai user interface, not even necessary to log on to the server.
